src/Components/BookCard/BookCard.js

```diff
@@ -13,9 +21,9 @@ const BookCard = ({ toReadList, haveReadList, bookInfo, addToToReadList, addToHa
           <h5 className='rank'>Rank: {bookInfo.rank}</h5>
         </div>
         <div className='book-btns'>
-          {!toReadList.includes(bookInfo) &&
+          {foundBook === undefined &&
```

Comment:
I like the conditional logic here to render the buttons. This serves as a great visual cue to the user & also prevents duplicates in each list